src/main/java/org/apache/sysml/api/mlcontext/ScriptExecutor.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.sysml.api.DMLScript;
 import org.apache.sysml.api.ScriptExecutorUtils;
+import org.apache.sysml.api.DMLScript.DMLOptions;
```

Comment:
Because of this weird dependency on DMLScript, should we consider moving DMLOptions into its own class?
Weirdly, I had thought of doing that, but since the Options are closely tied with parsing CLI, it didn't seem to make sense back then. But looking at this `import` statement, I am beginning to reconsider.